setup.py

```diff
@@ -60,9 +60,13 @@ def run(self):
         strip_all_hints(STRIP_DIRS)
         build_py.run(self)
 
+about = {}
+with open(join(this_directory, 'src', 'citrine', '__version__.py'), 'r') as f:
+    exec(f.read(), about)
```

Comment:
Very similar pattern to https://github.com/psf/requests/blob/master/setup.py#L60